session.rst

```diff
@@ -149,6 +149,9 @@ if you type-hint an argument with :class:`Symfony\\Component\\HttpFoundation\\Re
         public function __construct(RequestStack $requestStack)
         {
             $this->requestStack = $requestStack;
+            
+            // This does *not* work, since you cannot access the session inside a constructor
```

Comment:
In theory one can, but should not.

Reply:
I became aware of this cause phpstan crashes (see linked issue above).
So do you suggest a rewording like "is not recommended, since it might lead to unexpected results..."?

Reply:
It is indeed not recommended to fetch data in constructor.
Because calling a constructor is not a guarantee that the data will be actually processed in another method call, and also because the instance could change in between leading to unwanted/hard to debug side effects.
Same form `$requestStack->getRequest()` or `$securityHelper->getUser()`, ...
This is in line with services that should remain stateless.

Reply:
I reworded it.